pkg/metrics/grafana/tidb.json

```diff
@@ -20,7 +20,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "7.5.11"
+      "version": "7.5.17"
```

Comment:
can we revert those unrelated change, only copy you own panel?

Reply:
see this for an example https://github.com/pingcap/tidb/pull/54268/files#diff-e1c40eabcea75f285db03b26d672c728db9d6e797edc0d74f7e56af87066bf9aR13905

Reply:
Done